extensions/jackson/runtime/src/main/java/io/quarkus/jackson/runtime/vertx/QuarkusJacksonJsonCodec.java

```diff
@@ -72,7 +72,11 @@
         mapper.registerModule(module);
     }
 
-    private ObjectMapper prettyMapper() {
+    public ObjectMapper mapper() {
```

Comment:
How do you plan to call these methods?

Reply:
@cescoffier We currently have something like this in a class extending `QuarkusTestResourceLifecycleManager`:

```
ObjectMapper mapper = DatabindCodec.mapper();
mapper.configure(FAIL_ON_UNKNOWN_PROPERTIES, false);
```

After the Quarkus 2 migration, I'd like to change it to:

```
ObjectMapper mapper = QuarkusJacksonJsonCodec.mapper();
mapper.configure(FAIL_ON_UNKNOWN_PROPERTIES, false);
```

Reply:
ok, but the method must become static, no?

Reply:
Good catch, I forgot to make it static. Sorry about that.

Reply:
That's fixed now.

Reply:
Now it makes a lot more sense! We could use Thant new API to get the mapper in the vertx extension.